aries_vcx/src/handlers/util.rs

```diff
@@ -1,7 +1,7 @@
 use messages::{
     msg_fields::protocols::{
         connection::{invitation::Invitation, Connection},
-        cred_issuance::CredentialIssuance,
+        cred_issuance::{v2::CredentialIssuanceV2, CredentialIssuance},
```

Comment:
Please gather the V1 stuff in their own directory too. I didn't implicitly do it because we only had one version for each protocol and separating versions was not a problem.

Now, however, it's easier to figure out where to look for a specific version if they are separated in different modules.